### PR TITLE
Add current user badge on user listing page

### PR DESCRIPTION
**Pull request by yuxitang-amzn**
*Issue #, if available:*

*Description of changes:*
Add current user badge on user listing page.

Mockup:
image

Implementation:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/panels/user-list.tsx [hunk 1]
```diff
@@ -55,42 +57,54 @@ function dictView() {
...
   const [currentUsername, setCurrentUsername] = useState('');
-  // TODO: fetch current username
+
   useEffect(() => {
     const fetchData = async () => {
       try {
-        setUserData(await getUserList(props.coreStart.http));
+        const userDataPromise = getUserList(props.coreStart.http);
+        setCurrentUsername((await getAuthInfo(props.coreStart.http)).user_name);
+        setUserData(await userDataPromise);
```
Comment (tianleh) [inline public/apps/configuration/panels/user-list.tsx:99]: Minor: line 97 can be merged into line 99 just as your original code. Any reason for the separation?

File: public/apps/configuration/panels/user-list.tsx [hunk 2]
```diff
@@ -55,42 +57,54 @@ function dictView() {
...
       } catch (e) {
         console.log(e);
         setErrorFlag(true);
       }
     };
 
     fetchData();
-  }, [props.coreStart.http]);
+  }, [props.coreStart.http]); // TODO: fetch current username
```
Comment (tianleh) [inline public/apps/configuration/panels/user-list.tsx:107]: Looks the TODO is out of dated.

File: public/apps/configuration/panels/user-list.tsx (hunk 1 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/user-list.tsx:99]: Trying to make 2 requests in parallel... though don't know how much time can be saved...

File: public/apps/configuration/panels/user-list.tsx (hunk 2 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/user-list.tsx:107]: Will remove.

File: public/apps/configuration/panels/user-list.tsx (hunk 1 above)
Comment (tianleh) [reply public/apps/configuration/panels/user-list.tsx:99]: Sure. Just want to make sure that it is intentional.

**Pull request merged**